packages/browser/README.md

```diff
@@ -185,6 +185,57 @@ inputs.isInputsLoaded.then(() => {
 });
 ```
 
+### evervault.reveal()
+
+Use [evervault.reveal](https://docs.evervault.com/products/inputs#reveal) to show encrypted card
+details to show your users their encrypted card details in plaintext in a secure iFrame hosted by
+Evervault. Before using Reveal you'll first have to create a Relay to decrypt the card details;
+Reveal expects to receive the card data from the Relay as a JSON object with the schema below.
+
+Note: It is important that the endpoint that you create sets the applicable CORS headers so that it
+can be accessed from the Inputs iFrame. Otherwise your requests will fail!
+
+```json
+{
+  "cardNumber": "string | number",
+  "cvv": "string | number",
+  "expiry": "string | number"
+}
+```
+
+Once you have your endpoint that returns the encrypted card data, you'll need to create an
+[Evervault Inbound Relay](https://docs.evervault.com/products/inbound-relay) that will decrypt the
+encrypted card data as is passes through it, before it gets to the iFrame. Once you have created
+your Relay you can add the component to your React app.
+
+```html
+<div id="reveal"></div>
+
+<script>
+  let request = new Request("https://secure.example.com", {
```

Comment:
Should we change this to something like `https://my-endpoint.relay.evervault.com` to indicate it should be the relay endpoint?

Reply:
Great idea!